### PR TITLE
Add Tax Calculation for Delaware

### PR DESCRIPTION
**Pull request by podkolzzzin**

### CODE REVIEW

**Review by YaroslavKSE — APPROVED** [review]
Nice code! Works good

File: TaxCalculationSample.UnitTests/TaxCalculatorTests.cs [hunk 1]
```diff
@@ -45,6 +46,8 @@ public class TaxCalculatorTests
   [InlineData("Montana", 200000, 50000)]
   [InlineData("Montana", 500000, 185000)]
   [InlineData("Montana", 1000000, 540000)]
+  [InlineData("Delaware", 100000, 22000)]
+  [InlineData("Delaware", 50000, 9900)]
```
Comment (Zakhar-Khmurych) [inline TaxCalculationSample.UnitTests/TaxCalculatorTests.cs:50]: [InlineData("South Cansas", 100000, null]

**Review by Zakhar-Khmurych — CHANGES_REQUESTED** [review]

**Comment by Zakhar-Khmurych** [comment]
Added test for nonexistent state

**Review by Zakhar-Khmurych — APPROVED** [review]

**Review by Zakhar-Khmurych — APPROVED** [review]

**Comment by Zakhar-Khmurych** [comment]
Accepted

**Review by Zakhar-Khmurych — APPROVED** [review]

**Pull request merged**